AlphaWallet/Foundation/Subscribable.swift

```diff
@@ -21,15 +21,15 @@ open class Subscribable<T>: Hashable {
     }
 
     private var _value: T?
-    private var _subscribers: ThreadSafeDictionary<SubscribableKey, Subscription> = .init()
+    private var _subscribers: [SubscribableKey: Subscription] = .init()
```

Comment:
Not specifically for this one or this PR. But I think our approach is wrong; we should look greatly reduce the usage of `Subscribable`. It's just too brittle. What do you think?

Reply:
yep, 100 percent agree, it causes a lot of issues, for large wallet it causes creating a lot of threads for synchronization, so yes, it's fragile too much. Do you suggest using somethig instead? `PassthroughSubject<T, Failure> ``CurrentValueSubject<T, Failure>`

Reply:
Maybe just trigger it less granularly so ultimately whole screens refresh. Should be easier when we have more dumb objects rather than smarter ones like native Realms. Then we might even do diffs. Screens like UITableViews and UICollectionViews are only access in 1 (or a few, for pre-fetches) pages anyway. It will be way easier to reason. A good guide is how much we can reason about the code, eg. when we have a bug, can we often figure it out without needing to run the app to reproduce it with breakpoints?